linux-anvil-cuda/Dockerfile

```diff
@@ -26,7 +26,7 @@ ENV CUDA_HOME /usr/local/cuda
 
 # bust the docker cache so that we always rerun the installs below
 # ADD https://loripsum.net/api /opt/docker/etc/gibberish
-ADD https://worldtimeapi.org/api/timezone/Etc/UTC /opt/docker/etc/gibberish
+# ADD https://worldtimeapi.org/api/timezone/Etc/UTC /opt/docker/etc/gibberish
```

Comment:
FWIW, https://worldtimeapi.org/api/timezone/Etc/UTC is offline now. @beckermr found this other one https://www.random.org/cgi-bin/randbyte?nbytes=4096&format=h